frontend/src/components/commons/ContentContainer.jsx

```diff
@@ -4,7 +4,7 @@ import mini_banner from "./../../images/mini_banner.png";
 
 const styles = {
   container: {
-    maxWidth: 1140,
+    maxWidth: 1000,
```

Comment:
@joeyhua This is a significant change, and changes all your further designs. I don't think we should do this without a discussion and a really good reason.

Reply:
If the "back to top" button overlaps text, we should be making it have a white background so that it's always visible, not changing the width of the entire site.

Reply:
@fnormand01 instead of changing the width of the content container (which changes the width of everything on the site, can we change the width of just the text on the pre-instructions pages?

Reply:
Newest thought: we can put this back to 1140, and address it in another PR? I have some ideas, and my design pairing time with Joey is this afternoon.

Reply:
Put **maxWidth** back to 1140 in commit #https://github.com/code-for-canada/project-thundercat/pull/91/commits/71158457ada966f299c6cad4383b69977b6d3919